pkg/syncer/syncer.go

```diff
@@ -198,6 +198,7 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 	dnsIP := ""
 	// Get the DNS IP. The DNS service cannot be recreated as existing dnsConfig won't be updated.
 	err = wait.PollImmediate(2*time.Second, time.Minute, func() (bool, error) {
+		logger.Info(fmt.Sprintf("performing a DNS lookup of %s", cfg.DNSServer))
```

Comment:
To be consistent with the structural logging changes (ref #2134) this should probably either use `WithValues`, or pass multiple args to the `Info` instead of using `Sprintf`?